templates/oam/workloads/1.10.0/destinationrule.istio.meshery.layer5.io_definition.json

```diff
@@ -0,0 +1,20 @@
+{
+ "kind": "WorkloadDefinition",
+ "apiVersion": "core.oam.dev/v1alpha1",
+ "metadata": {
+  "name": "DestinationRule.Istio",
+  "creationTimestamp": null
+ },
+ "spec": {
+  "definitionRef": {
+   "name": "destinationrule.Istio.meshery.layer5.io"
```

Comment:
Inconsistency in casing here. Should component be lowercase?

Reply:
No this is fine. This is just to generate the file name. Plus this is consistent with metadata.name.

Reply:
ok